Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -1730,3 +1730,111 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Pull iTunes deltas from Collection
     C4Log("-------- %.3f sec with deltas, %.3f sec without; %.2fx speed",
           timeWithDelta, timeWithoutDelta, timeWithoutDelta/timeWithDelta);
 }
+
+// This test may be used in future, if kDefaultMaxHistory > kDefaultMaxRevTreeDepth, and we wish to test
+// the conflict resolution behaviour for randomly generated rev history.
+#if 0
```

Comment:
Why disable this test? It is still valuable to test sync in the presence of gaps, IMHO.